test/upgrade/mzcompose.py

```diff
@@ -137,5 +181,192 @@ def test_upgrade_from_version(
         temp_dir,
         seed,
         "--validate-catalog=/share/mzdata/catalog",
-        f"check-from-{version_glob}-{filter}.td",
+        f"check-{style}from-{version_glob}-{filter}.td",
+    )
+
+
+def workflow_ssl(c: Composition, parser: WorkflowArgumentParser) -> None:
+    """Test upgrades from various versions."""
+
+    parser.add_argument(
```

Comment:
There is a bit of a code duplication with parsing the same option in the same way at two locations in the script.

Reply:
I think this entire method was meant to be deleted? `workflow_default` seems to handle it all.

Reply:
yah I forgot to delete this